src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -113,8 +113,14 @@ def _polish_bad_errors(ex):
 
 # end of the new ones #
 # beginning of the old ones ###
+
+# in June, remove all olds stuff below and replace with one following line
+# cli_commands_removed('appservice web', "'az appservice web' is forever gone. You should use 'az webapp'")
```

Comment:
My suggestion is to still register the command with a handler that only prints the "this command has been removed" message on execution. This way, we don't need another module global set dictionary of things to remove.